prince/README.md

```diff
@@ -121,3 +121,48 @@ GPU name: Tesla V100-SXM2-32GB-LS
 CPU matmul elapsed: 1.1984939575195312 sec.
 GPU matmul elapsed: 0.01778721809387207 sec.
 ```
+
+# Mujoco Installation on Prince
+
+This tutorial assumes that you have already installec conda in your scratch folder. Please note that your conda environment must be stored in $SCRATCH not in $HOME directory. You will get weird buildlock errors otherwise !
+
+We also assume that you have put the Mujoco200 and the license key folder inside ~/.mujoco/ folder.
+
+```bash
+> conda create -n test_mujoco python=3.6
+
+>  singularity exec /beegfs/work/public/singularity/mujoco-200.sif /bin/bash
+
+> source activate test_mujoco
```

Comment:
```suggestion
> conda activate test_mujoco
```

in general its easier to use `conda activate` since then you can use `conda deactivate` (`source deactivate` does not exist)